chaos.c

```diff
@@ -9,7 +9,7 @@
 #include <unistd.h>
 #include <string.h>
 #include <sys/un.h>
-#if defined(__HAIKU__) || defined(__OpenBSD__)|| defined(__NetBSD__)
```

Comment:
I'd like the pull request to be refactored a bit so that changes like this that revise previous changes, are folded back into those commits.

Reply:
This commit is the most obvious one, but there are bits and pieces here and there that also apply.

Reply:
Later OS support is all kind of intertwined at this point now, so I'll probably just squash it down into one commit without cosmetic changes.